packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -538,20 +538,13 @@ export const cliOptionsConfig: VitestCLIOptions = {
   cache: {
     description: 'Enable cache',
     argument: '', // allow only boolean
-    subcommands: {
-      dir: {
-        description: 'Path to the cache directory',
-        argument: '<path>',
-        normalize: true,
-      },
-    },
     // cache can only be "false" or an object
     transform(cache) {
       if (cache)
         return {}
       return cache
     },
-  },
+  } as VitestCLIOptions['cache'],
```

Comment:
No, you need to manually pass `null` to `dir` subcommand instead of removing it

Reply:
Thanks for letting me know, I've updated it.